samples/server/petstore/java-play-framework-no-swagger-ui/app/Module.java

```diff
@@ -0,0 +1,13 @@
+import com.google.inject.AbstractModule;
+
+import controllers.*;
+
+public class Module extends AbstractModule {
+
+    @Override
+    protected void configure() {
+        bind(PetApiControllerImpInterface.class).to(PetApiControllerImp.class);
```

Comment:
In this example, the interface exists. This is the "no-swagger-ui" example, so only the UI is absent.